apps/frontend/src/components/form/GrantForm.tsx

```diff
@@ -47,237 +49,278 @@ const alertOnSubmit = async (values: FormValues) => {
-      {({ field, form }: FieldProps) => (
-        <FormControl
-          isInvalid={Boolean(
-            form.errors['addressState'] && form.touched['addressState']
-          )}
-        >
-          <FormLabel htmlFor={'addressState'}>State</FormLabel>
-          <Input {...field} id={'addressState'} />
-        </FormControl>
-      )}
-    </Field>
-
-    <Field name="addressZipcode">
-      {({ field, form }: FieldProps) => (
-        <FormControl
-          isInvalid={Boolean(
-            form.errors['addressZipcode'] && form.touched['addressZipcode']
-          )}
-        >
-          <FormLabel htmlFor={'addressZipcode'}>Zip Code</FormLabel>
-          <NumberInput>
-            <NumberInputField {...field} id={'addressZipcode'} />
-          </NumberInput>
-        </FormControl>
-      )}
-    </Field>
-    <Field name="phoneNumber">
-      {({ field, form }: FieldProps) => (
-        <FormControl
-          isRequired
-          isInvalid={Boolean(
-            form.errors['phoneNumber'] && form.touched['phoneNumber']
-          )}
-        >
-          <FormLabel htmlFor={'phoneNumber'}>Phone Number</FormLabel>
-          <InputGroup>
-            <InputLeftElement
-              pointerEvents="none"
-              children={<PhoneIcon color="gray.300" />}
-            />
-            <Input type="tel" />
-          </InputGroup>
-        </FormControl>
-      )}
-    </Field>
-    <Field name="cellPhoneNumber">
-      {({ field, form }: FieldProps) => (
-        <FormControl
-          isInvalid={Boolean(
-            form.errors['cellPhoneNumber'] && form.touched['cellPhoneNumber']
-          )}
-        >
-          <FormLabel htmlFor={'cellPhoneNumber'}>Cell Phone Number</FormLabel>
-          <InputGroup>
-            <InputLeftElement
-              pointerEvents="none"
-              children={<PhoneIcon color="gray.300" />}
-            />
-            <Input type="tel" />
-          </InputGroup>
-        </FormControl>
-      )}
-    </Field>
-    <Field name="email">
-      {({ field, form }: FieldProps) => (
-        <FormControl
-          isRequired
-          isInvalid={Boolean(form.errors['email'] && form.touched['email'])}
-        >
-          <FormLabel htmlFor={'email'}>Email Address</FormLabel>
-          <Input {...field} type="email" id={'email'} />
-        </FormControl>
-      )}
-    </Field>
-    <Field name="requestedGrantAmount">
-      {({ field, form }: FieldProps) => (
-        <FormControl
-          isInvalid={Boolean(
-            form.errors['requestedGrantAmount'] &&
-              form.touched['requestedGrantAmount']
-          )}
-        >
-          <FormLabel htmlFor={'requestedGrantAmount'}>
-            Requested Grant Amount (in USD)
-          </FormLabel>
-          <InputGroup>
-            <InputLeftElement
-              pointerEvents="none"
-              color="gray.300"
-              fontSize="1.2em"
-              children="$"
-            />
+    <Box py="2">
```

Comment:
looks like "py" stands for "paddingY" as a chakra ui prop, could you change it to the full written out version just for readability?

Reply:
Yes!